### PR TITLE
BUG CICD PIPELINE issues

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes # .

#### Changes proposed in this Pull Request:
- 
- 
- 

#### Screenshots / Screen Recording (if applicable)
-

#### To Test:
- 

#### Any specific feedback you are looking for?
-

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -223,6 +224,9 @@ jobs:
                   command: |
                       cd demos/showcase
                       yarn prettier:check
+            - run:
+                  name: 'Link Components'
+                  command: yarn link:components
```
Comment (joebochill) [inline .circleci/config.yml:229]: Is this needed? Installing the local version of components is already handled on line 216.

Comment (ArshdeepSingh3-eaton) [reply .circleci/config.yml:229]: it was simply saying the files are not found and failing the pipeline

Comment (JeffGreiner-eaton) [reply .circleci/config.yml:229]: Can you confirm what files from circleCI build job. I only have seen snapshot tests failing

Comment (joebochill) [reply .circleci/config.yml:229]: I'm curious about this too...the build folder should be there.

We build the components in the build_library job and then pack the dist folder into a tgz, which we then preserve in the workspace for the subsequent job:
image

And then we install the components into the showcase from that packaged version:
image

Unless the relative path got messed up, the files should be there.

Comment (ArshdeepSingh3-eaton) [reply .circleci/config.yml]: the component Package.json did not include md3 folder so the npm-pack was leaving md3 folder out of this and creating those errors
Fixed that

File: components/package.json [hunk 2]
```diff
@@ -83,7 +83,8 @@
         "README.md",
         "CHANGELOG.md",
         "core/",
-        "themed/"
+        "themed/",
+        "md3/"
```
Comment (joebochill) [inline components/package.json:87]: Aha! good catch!

**Review by joebochill — APPROVED** [review]

**Pull request merged**